garrysmod/lua/includes/modules/cvars.lua

```diff
@@ -18,111 +11,124 @@ module( "cvars" )
 
 local ConVars = {}
 
-function GetConVarCallbacks( name, CreateIfNotFound )
-
-	local Tab = ConVars[ name ]
+--[[---------------------------------------------------------
+	Name: GetConVarCallbacks
+	Desc: Returns a table of the given ConVars callbacks
+-----------------------------------------------------------]]
+function GetConVarCallbacks( name, createIfNotFound )
 
-	if ( CreateIfNotFound && !Tab ) then
-		Tab = {}
-		ConVars[ name ] = Tab
+	local tab = ConVars[ name ]
+	if ( createIfNotFound and !tab ) then
+		tab = {}; ConVars[ name ] = tab
 	end
 
-	return Tab
+	return tab
+
 end
 
 --[[---------------------------------------------------------
 	Name: OnConVarChanged
 	Desc: Called by the engine
 -----------------------------------------------------------]]
-function OnConVarChanged( name, oldvalue, newvalue )
+function OnConVarChanged( name, old, new )
 
-	local Callbacks = GetConVarCallbacks( name )
-	if ( !Callbacks ) then return end
+	local tab = GetConVarCallbacks( name )
+	if ( !tab ) then return end
 
-	for k, v in pairs( Callbacks ) do
-
-		if ( istable( v ) ) then
-			v[ 1 ]( name, oldvalue, newvalue )
+	for i = 1, #tab do
+		local callback = tab[ i ]
+		if ( type( callback ) == "table" ) then

```

Comment:
What's the point of changing `istable( v )` calls  to `type( callback ) == "table"`?

Reply:
purpose is to make the code shorter, why do we need to localize `istable` if we already have `type`, why do we need both functions?

Reply:
Okay